tools/go-cli/go-whisk-cli/commands/action.go

```diff
@@ -621,13 +603,19 @@ func parseAction(cmd *cobra.Command, args []string) (*whisk.Action, bool, error)
         } else if flags.action.kind == "nodejs:6" || flags.action.kind == "nodejs:6.0" ||
             flags.action.kind == "nodejs:6.0.0" {
             action.Exec.Kind = "nodejs:6"
-        } else if matched, _ := regexp.MatchString(".swift$", stat.Name()); matched {
+        } else if len(flags.action.kind) > 0 {
```

Comment:
It seems that python and java runtime are supported in the code below that examines the extension (.py, .jar).  Should there be --kind values of `python` and `java`?